files/en-us/web/api/xrsession/environmentblendmode/index.html

```diff
@@ -19,37 +19,30 @@
 ---
 <div>{{APIRef("WebXR Device API")}}</div>
 
-<p><span class="seoSummary">The {{domxref("XRSession")}} interface's
-    <em>read-only</em> <code><strong>environmentBlendMode</strong></code> property
-    identifies if—and to  what degree—the computer-generated imagery is overlaid atop the
-    real world.</span> This is used to differentiate between fully-immersive VR sessions
-  and AR sessions which render over a pass-through image of the real world, possibly
-  partially transparently.</p>
-
-<p>The value is a {{domxref("DOMString")}} which contains one of the values defined by the
-  {{domxref("XREnvironmentBlendMode")}} enumerated type.</p>
-
-<div class="notecard warning">
-  <p><strong>Important:</strong> <code>environmentBlendMode</code> is part of the <a
-      href="https://immersive-web.github.io/webxr-ar-module/">WebXR Augmented Reality
-      Module</a>, which has not yet reached a stable state. You should use caution when
-    using its features.</p>
-</div>
+<p>The {{domxref("XRSession")}} interface's <em>read-only</em> <code><strong>environmentBlendMode</strong></code>
+property identifies if—and to  what degree—the computer-generated imagery is overlaid atop the real world.</p>
```

Comment:
```suggestion
property identifies if, and to what degree, the computer-generated imagery is overlaid atop the real world.</p>
```
There is an extra space. 

I'm not a fan of parenthesis _en_ dash, so I used commas instead, which I find better as there is no commas in the clause. I wouldn't have done this if there weren't anything else to fix here.